pom.xml

```diff
@@ -180,14 +193,35 @@
                     </includes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.2.1</version>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <relocations>
+                                <relocation>
+                                    <pattern>io.vavr</pattern>
+                                    <shadedPattern>io.shaded.vavr</shadedPattern>
```

Comment:
how about prefixing to avoid possible conflicts?

something like com.mercateo.spring.security.jwt.relocated.io.vavr or something?

Reply:
done